momento-sdk/src/main/java/momento/sdk/messages/CacheGetResponse.java

```diff
@@ -26,41 +23,13 @@ public byte[] valueByteArray() {
       return value.toByteArray();
     }
 
-    /**
-     * Gets the retrieved value as a {@link ByteBuffer}.
-     *
-     * @return the value.
-     */
-    public ByteBuffer valueByteBuffer() {
-      return value.asReadOnlyByteBuffer();
-    }
```

Comment:
This one is the only one I'll miss _a little_. It could be nice to have a zero-copy way to get the bytes, but I'm +1 on waiting for users to express an opinion on it!